Lib/glyphsLib/builder/glyph.py

```diff
@@ -270,6 +270,8 @@ def to_ufo_glyph_height_and_vertical_origin(self, ufo_glyph, layer):
 
     if layer.vertWidth:
         ufo_glyph.height = layer.vertWidth
+    elif "rotat" in ufo_glyph.name:
```

Comment:
I've never heard of this "rotat" glyph name suffix. Is that your particular glyph naming scheme, or some new magic feature in Glyphs.app? I sincerely hope that's not the latter.

Reply:
From what I can tell, it is a common suffix for Japanese development using the AFDKO (Fontworks has it in its AFDKO sources). Yes, it does have 'magic feature' in Glyphs, but I do not believe it is exclusive to Glyphs.

Reply:
Furthermore, given that I've been reviewing fonts from 17 different foundries, and they have all used it when indicating rotated forms gives me confidence that it can be explicitly used :).

Reply:
can't the font developer simply set the vertWidth explicitly for these glyphs instead of relying on a magic suffix?

Reply:
I've provided you an example of a font that doesn't have vertWidth set, that clearly wants the font to be used for vertical typesetting, and has followed (what appears to be) a common convention for glyph naming in the Japanese type design field.

All this code does is provide a safety net in case a designer does not set that field. It is really no different than setting it to `ascender - descender`. Do you deliberately want them to produce bad fonts?

Reply:
is this documented anywhere? How can we glyphsLib devs know that? The job of the tool is to try mimic what Glyphs.app does. If we can get confirmation from some authoritative source that that's indeed the case, we will certainly do what needs to be done (even though I do confess I personally dislike this kind of automatic features).

Reply:
TBH, I see this more as a fallback safety net than an automatic feature that someone would want to use—it'll always be better to set these values manually. 

As for the documentation, I haven't seen any but what I have found in my own testing. One thing I did locate was this comment which seems to indicate the behavior I suggest: 

> If the glyph has a suffix “.rotat”, “.vert” or “.vrt2” and contains a component that is rotated by 90° the automatic alignment will position the component and set the vertWidth correctly.

https://forum.glyphsapp.com/t/how-to-make-rotat-glyphs-in-japanese-fonts/13014/4

But probably best to wait for Georg to confirm.